src/jit/lower.cpp

```diff
@@ -1423,13 +1512,15 @@ void Lowering::LowerCall(GenTree* node)
         BlockRange().InsertBefore(insertionPoint, std::move(resultRange));
 
         call->gtControlExpr = result;
+        ContainCheckRange(std::move(resultRange));
```

Comment:
This pattern--inserting a range and then running the contain check--appears in multiple places in this PR. The comment above applies to all of these occurrences.